features/fee_calculator/litigator/interim_fee_calculator.feature

```diff
@@ -177,3 +177,182 @@ Feature: litigator completes interim fee page using calculator
     And the interim fee should have its price_calculated value set to true
 
     And I eject the VCR cassette
+
+  @fee_calc_vcr
```

Comment:
Nit: Could we define the feature flag `@fee_calc_vcr` once on line #1

Reply:
Fixed 👍